sdk/plugin/grpc_backend_client.go

```diff
@@ -280,3 +280,14 @@ func (b *backendGRPCPluginClient) Type() logical.BackendType {
 
 	return logical.BackendType(reply.Type)
 }
+
+func (b *backendGRPCPluginClient) Version() logical.VersionInfo {
+	reply, err := b.client.Version(b.doneCtx, &pb.Empty{})
+	if err != nil {
+		b.Logger().Debug("Error getting plugin version; assuming method is missing", "err", err)
```

Comment:
Is there a specific error we can check for to determine that it's because the method was missing?